### PR TITLE
Adds layout section to README.

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -349,3 +349,47 @@ To generate an authentication token run RAILS_ENV=production bin/rails generate_
 ```
 {"invoked_by" => "workflow-service"}
 ```
+
+## File layout
+Objects created prior to August 2024 using the legacy, unversioned layout. Objects created thereafter have the new versioned layout.
```
Comment (lwrubel) [inline README.md:354]: Is something missing from this first sentence?

File: README.md [hunk 2]
```diff
@@ -349,3 +349,47 @@ To generate an authentication token run RAILS_ENV=production bin/rails generate_
...
+
+In a future workcycle, access applications (image server, streaming server, etc.) will be changed to utilize the versioned layout, at which point all objects can be migrated to the versioned layout.
+
+### Unversioned layout
+In PURL file system:
+```
+/purl/document_cache/bc/123/df/4567/
+  |   cocina.json
+  |   public <-- public xml
+  \-- meta.json
```
Comment (lwrubel) [inline README.md:364]: What does `\--` mean in this layout?

**Review by lwrubel — APPROVED** [review]
Approved pending consideration of questions/comments.

**Comment by lwrubel** [comment]
Thanks for adding this to the README. I know I'll be using this as a reference.

File: README.md (hunk 2 above)
Comment (justinlittman) [reply README.md:364]: The end of this directory. I took this from https://datatracker.ietf.org/doc/html/draft-kunze-bagit-10

File: README.md (hunk 1 above)
Comment (justinlittman) [reply README.md:354]: Missing proper grammar? Fixing.

File: README.md (hunk 2 above)
Comment (lwrubel) [reply README.md:364]: OK, thanks, that was new to me (but maybe not other people).

**Pull request merged**